src/librustc_const_eval/pattern.rs

```diff
@@ -280,7 +280,7 @@ impl<'a, 'tcx> Pattern<'tcx> {
         let mut pcx = PatternContext::new(tcx, param_env_and_substs, tables);
         let result = pcx.lower_pattern(pat);
         if !pcx.errors.is_empty() {
-            span_bug!(pat.span, "encountered errors lowering pattern: {:?}", pcx.errors)
+            debug!("encountered errors lowering pattern: {:?}", pcx.errors)
```

Comment:
Wait, can't this be `tcx.sess.delay_span_bug`? That would make sure we have some other error emitted.

Reply:
Yeah that worked. I am going to add a commit to replace `debug!()` with `delay_span_bug()`.